### PR TITLE
Move brendandburns to to emeritus status.

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Per note in https://github.com/kubernetes/kubernetes/issues/93637#issuecomment-668383474 @brendandburns has requested to be marked as emeritus.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
@brendanburns - Thank you for all your work over the years! Should you find the time again to dive into things, we'll be so happy to move you back.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/sig contributor-experience
/priority important-soon

/assign @brendandburns

### CODE REVIEW

**Comment by spiffxp** [comment]
@dims 
I think the verify failure is legitimate?  It appears to be having trouble with the `translation/OWNERS` removal.

(bindata is picking up way too much, it shouldn't need OWNERS, but that's orthogonal to this)

**Comment by dims** [comment]
@spiffxp thanks. fixed.

/assign @liggitt

File: translations/OWNERS [hunk 1]
```diff
@@ -1,6 +0,0 @@
-# See the OWNERS docs at https://go.k8s.io/owners
-
-reviewers:
```
Comment (liggitt) [inline translations/OWNERS:3]: needs rebasing on the "add sigs to root folders" commit, I think, and then bindata regenerated

**Comment by liggitt** [comment]
I wasn't sure if https://github.com/kubernetes/kubernetes/issues/93637#issuecomment-668383474 was referring to moving root approver to emeritus or dropping from review lists everywhere. once that is clarified and this is rebased, lgtm

**Comment by dims** [comment]
@liggitt @brendanburns updated this to the bare minimum which was the root OWNERS switch. PTAL.

File: OWNERS [hunk 2]
```diff
@@ -10,14 +10,14 @@ filters:
       - thockin
       - liggitt
     approvers:
-      - brendandburns
```
Comment (BenTheElder) [inline OWNERS:13]: should we have updated reviewers above as well?

**Comment by thockin** [comment]
Probably.  I don't think Brendan is active at this level any more.

**Pull request merged**